src/Features/Core/Portable/xlf/FeaturesResources.cs.xlf

```diff
@@ -2185,6 +2185,11 @@ Pozitivní kontrolní výrazy zpětného vyhledávání s nulovou délkou se obv
         <target state="new">Renaming a captured variable, from '{0}' to '{1}' requires restarting the application.</target>
         <note />
       </trans-unit>
+      <trans-unit id="Renaming_parameter_0_requires_restarting_the_application_because_it_is_not_supported_by_the_runtime">
+        <source>Renaming parameter {0} requires restarting the application because it is not supported by the runtime.</source>
```

Comment:
>{0} [](http://example.com/codeflow?start=35&length=3)

I believe the {0} argument is gonna be the symbol kind, in this case "parameter". So this message should be 

Renaming {0} requires restarting the application because it is not supported by the runtime